test/common/flags.test.ts

```diff
@@ -103,35 +126,41 @@ describe('requiredDoceOrgFlag', () => {
       isEnvVar: () => false,
     });
     try {
-      await Parser.parse(['--requiredDoceOrg='], {
+      await Parser.parse([`--requiredDoceOrg=${invalidAlias}`], {
         flags: { requiredDoceOrg: requiredDoceOrgFlag() },
       });
       assert.fail('This should have failed');
     } catch (err) {
       // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-      expect(err.message).to.include(
-        'You must specify the DevOps Center org username by indicating the -c flag on the command line or by setting the --target-devops-center configuration variable.'
-      );
+      expect(err.message).to.include(`No authorization information found for ${invalidAlias}`);
     }
   });
+});
 
-  it('fails when an invalid alias is provided and the --target-devops-center config var is not set', async () => {
-    const invalidAlias = 'invalidAlias';
-    sandbox.stub(ConfigAggregator.prototype, 'getInfo').returns({
-      value: null,
-      key: ConfigVars.TARGET_DEVOPS_CENTER,
-      isLocal: () => false,
-      isGlobal: () => true,
-      isEnvVar: () => false,
-    });
+describe('waitFlag', () => {
```

Comment:
Can we have a test to assert that --wait and --async flags are mutully exclusive?